libs/confman/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ target_include_directories(confman
 target_link_libraries(confman
   PUBLIC
     Boost::boost
+    Boost::uuid
```

Comment:
use similar uuid as in item.hpp

Reply:
tried to mimick item.hpp, please see changes.